### ISSUE TITLE
assignUserRoles not mapping to objects correctly.

### ISSUE THREAD

**Issue by dbarkowsky** [bcgov/citz-imb-sso-css-api#2]
The role names were being mapped incorrectly. The map didn't return an object for each element because the body of the map function was a function, not an object. Hard to describe, but here's some updated code:

``` ts
export const assignUserRoles = async (
  username: string,
  roleNames: string[]
) => {
  if (DEBUG) logDebug("assignUserRoles");
  return await request({
    integrationEndpoint: true,
    endpoint: `users/${username}/roles`,
    method: "POST",
    body:
      (roleNames.map((roleName) => ({
        name: roleName,
      })) as RequestBody) ?? [],
  });
};
```

**Comment by dbarkowsky** [comment]
Fixed with this PR: https://github.com/bcgov/citz-imb-kc-css-api/pull/4

**Comment by BradyMitch** [comment]
Resolved in `1.3.1`
